src/MeetingsSDKAdapter.js

```diff
@@ -41,6 +41,19 @@ import {
  * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L58}
  */
 
+/**
+ * Enum for meeting local share state.
+ *
+ * @readonly
+ * @enum {string}
+ */
+export const MeetingLocalShareState = {
```

Comment:
will be imported from interfaces, see https://github.com/webex/component-adapter-interfaces/pull/67